actions/clean-workspace/action.yml

```diff
@@ -9,7 +15,10 @@ runs:
         rm -f config/keystore.jks
```

Comment:
The workspace used for running the pipeline doesn't get reused ([unless specified directly via caching](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/caching-dependencies-to-speed-up-workflows#about-caching-workflow-dependencies)), making any sort of cleanup pointless